### DISCUSSION TITLE
can we improve swap effect by using more photos?

### DISCUSSION THREAD

**Discussion by mokerbay** [hacksider/Deep-Live-Cam#475] [Ideas]
I tried this for hours and found that the effect is not stable. sometimes it works well(with Elon,or other one's face that shape fits my face), but sometimes not that good(with olderman like Trump,Biden,or faces that too thin to fit mine)
so can we just put more photo to extract more details like wrinkles so it'll be more realistic

**Comment by hacksider** [comment]
I really can't answer your question as the model is just third party and we're just using it. What I think you're thinking is the use of dfm files using deep face live. Surely you can do that using that software ☺️
